packaging/packager_pip.sh

```diff
@@ -15,6 +15,8 @@ if [[ $python_version == '3.9' ]]; then export DISABLE_AUTOPY=1; fi
 
 # environment dependencies not provided by pip
 # python3
+if [[ $python_version == '3.11' ]]; then dnf -y install python3.11 python39-devel; fi
+if [[ $python_version == '3.10.8' ]]; then dnf -y install python3.10.8 python39-devel; fi
```

Comment:
This does not follow the logic of the lines below, the versions see right below.

Reply:
Okay, onto it

Reply:
Ok removed so resolving